.github/ISSUE_TEMPLATE/docs_problem.yml

```diff
@@ -5,7 +5,7 @@ body:
   - type: markdown
     attributes:
       value: |
-        Hi, thank you for taking the time to create an issue! If you're asking for _new_ docs, or have a suggestion on how to enhance existing docs, please open [a docs suggestion](https://github.com/directus/directus/discussions/new?category=docs-suggestions) instead!
+        Hi, thank you for taking the time to create an issue! If you're asking for _new_ docs, or have a suggestion on how to enhance existing docs, please open [a docs suggestion](https://github.com/directus/directus/discussions/new?category=docs_suggestions) instead!
```

Comment:
The `docs_suggestions` category looks to be non-existent. 👀 

<img width="489" alt="image" src="https://github.com/directus/directus/assets/26413686/4264f39d-4849-40ce-b8b9-39c38f257589">

Ref: https://github.com/directus/directus/discussions/new?category=docs_suggestions

Reply:
Ah... The URL should be updated to issues instead of the current discussions.

```suggestion
        Hi, thank you for taking the time to create an issue! If you're asking for _new_ docs, or have a suggestion on how to enhance existing docs, please open [a docs suggestion](https://github.com/directus/directus/issues/new?labels=Documentation&template=docs_suggestions.yml) instead!
```

On a similar note, the link in `.github/ISSUE_TEMPLATE/docs_suggestions.yml` should also be updated to point to issues too.

Reply:
Thanks ❤️ Somehow, it seemed to work 😅 Will update the other link too 👍